webpack.config.babel.js

```diff
@@ -4,7 +4,7 @@ const StyleLintPlugin = require('stylelint-webpack-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
 const VisualizerPlugin = require('webpack-visualizer-plugin2');
 
-const MAX_DEVELOPMENT_OUTPUT_SIZE = 3300000;
+const MAX_DEVELOPMENT_OUTPUT_SIZE = 3400000;
```

Comment:
How come? 🤔 

<img width="144" alt="obrazek" src="https://github.com/user-attachments/assets/925a2602-0935-4b16-ad09-8c608f01775b" />

Reply:
There is no direct relationship between lines changed and bundle size. For example bundle size is not effected by comments and such.